src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return first => {
+    count++;
+
+    return second => {
+      if (count > 3 && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return first + second;
+      }
```

Comment:
`else` with returns is redundant
```suggestion
      if (count > 3 && count % 2 === 0) {
        return 'Bzzz... Error!';
      }
      
      return first + second;
```